doc/how_to_build_win.md

```diff
@@ -23,26 +23,26 @@ This software can be built using Visual Studio 2015 and Qt 5.9
   - `git config core.safecrlf true`
 
 ## Installation of Required Libraries
-Because of the size of these libraries, they are not maintained in the git repository.
-They will have to be installed seperately as follows.
+Because of the size of these libraries, they are not maintained in the git repository. They will have to be installed seperately as follows.
 
 ### `lib` and `dll`
 - `lib` and `dll` files are tracked by [Git Large File Storage](https://git-lfs.github.com/).
 - Execute `git lfs pull` after `git clone` by using the `lfs` client.
+Note: Also all of these actions can also be done using the GitHub Desktop Client.  Recommended if you are not proficient with git commands.  https://desktop.github.com/
```

Comment:
```suggestion
You can also perform these actions using GitHub for Desktop Client.
```

I think the general comment is fine, but programmers will know how to use a CLI.

Reply:
Thanks for the changes

Reply:
Thanks for the changes